sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -313,11 +313,96 @@ for (int i = 0; i < receiptPageResults.size(); i++) {
 }
 ```
 
+### Recognize business cards
+Recognize data from business cards using a prebuilt model. Business card fields recognized by the service
+can be found [here][service_recognize_business_card].
```

Comment:
I think it will be better if we point to https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/concept-business-cards?#fields-extracted 
I will expect this page to get updated instead of us updating the reference of the link every time there is a new preview/stable version

Reply:
I can do that. Also that means `https://aka.ms/formrecognizer/businesscardfields` is useless and can be removed.

Reply:
not really. In case we ever have to change where the link is pointing, it is better to change it just in one place than go through the code and updating it.
I will say leave it as is after the change you did of redirecting it to the docs place

Reply:
After I revisited, i found the link is used all over the place in the javadoc. I would probably keep it as it is. Since it is short.

Reply:
Yes please leave it. I am not advocating for deleting it. My initial ask was to update where the link was pointing too (which you already did) but never to delete it :)